repo/datastore.go

```diff
@@ -13,24 +14,31 @@ import (
 
 type Datastore interface {
 	Config() Config
-	Followers() Followers
-	Following() Following
-	OfflineMessages() OfflineMessages
-	Pointers() Pointers
-	Settings() Settings
-	Inventory() Inventory
-	Purchases() Purchases
-	Sales() Sales
-	Cases() Cases
-	Chat() Chat
-	Notifications() Notifications
-	Coupons() Coupons
-	TxMetadata() TxMetadata
-	ModeratedStores() ModeratedStores
+	Followers() FollowerStore
+	Following() FollowingStore
+	OfflineMessages() OfflineMessageStore
+	Pointers() PointerStore
+	Settings() ConfigurationStore
+	Inventory() InventoryStore
+	Purchases() PurchaseStore
+	Sales() SaleStore
+	Cases() CaseStore
+	Chat() ChatStore
+	Notifications() NotificationStore
+	Coupons() CouponStore
+	TxMetadata() TransactionMetadataStore
+	ModeratedStores() ModeratedStore
```

Comment:
These names are up for discussion, but typically I try to call Interfaces a verb-like name to indicate the sort of actions it can take. Unfortunately, because these are interfaces for exactly one type of object (the interface isn't shared across different structs) the name for these end up being a little more specific indicating the type of Store they are. Naming things is hard.